modules/claims_api/app/serializers/claims_api/power_of_attorney_serializer.rb

```diff
@@ -2,6 +2,6 @@
 
 module ClaimsApi
   class PowerOfAttorneySerializer < ActiveModel::Serializer
-    attributes :status, :date_request_accepted, :representative, :veteran, :previous_poa
```

Comment:
removing `veteran since the only thing being provided appears to be the participant ID